demos/dermatology.html

```diff
@@ -134,13 +134,16 @@
         if (def) {
             def.split("&").forEach(function (param, idx) {
                 var paramParts = param.split("=");
+                if (paramParts[0] === "hpcc_debug") { return; }

```

Comment:
Are you forcing this to be the first param?  I would have though it could be a param at any position and easiest to have it as the last (from a typing it in point of view...